.github/workflows/update-gardenlogin.yaml

```diff
@@ -0,0 +1,34 @@
+name: gardenlogin-updater
+
+on:
+  release:
+    types:
+      - published
+jobs:
+  update_gardenlogin_in_homebrew_tap:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the binary-files
+        id: build_binary_files
+        run: |
+          sudo apt-get update
+          sudo apt-get install make -y
+          make build
+          echo ::set-output name=latest_release_filtered_tag::${GITHUB_REF##*/}
+      - name: Upload binaries to release
+        uses: AButler/upload-release-assets@v2.0
+        with:
+          files: 'bin/darwin-amd64/gardenlogin_darwin_amd64;bin/linux-amd64/gardenlogin_linux_amd64'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          release-tag: ${{ steps.build_binary_files.outputs.latest_release_filtered_tag }}
+      - name: Send update with latest versions to gardener/homebrew-tap
+        run: |
+          mac_sha256sum=$(sha256sum bin/darwin-amd64/gardenlogin_darwin_amd64 | awk '{print $1}')
```

Comment:
Since I don't have any real feedback, take this instead: Can we call this `darwin_sha256sum` instead? The filename contains "..._darwin_..." and reading "mac_sha256" makes me think of HMAC's at first.

Reply:
done